energy/jsonld-contexts/energy.jsonld

```diff
@@ -0,0 +1,28 @@
+{
+    "@context": {
+        "Building": "https://vocab.egm.io/Building",
+        "Client": "https://vocab.egm.io/Client",
+        "Contract": "https://vocab.egm.io/Contract",
+        "DeliveryPoint": "https://vocab.egm.io/DeliveryPoint",
+        "Invoice": "https://vocab.egm.io/Invoice",
+        "Site": "https://vocab.egm.io/Site",
+        "Supplier": "https://vocab.egm.io/Supplier",
+        "amount": "https://vocab.egm.io/amount",
+        "city": "https://vocab.egm.io/city",
```

Comment:
for general things like `city`, `postalCode`, ... have a look at schema.org

Reply:
Found for postalCode but not for city surprisingly. Has been modified

Reply:
there is https://schema.org/City (but it's more like an entity than a property)